### ISSUE TITLE
Relooper bootstrap race condition

### ISSUE THREAD

**Issue by kripken** [kripken/emscripten#773]
The relooper is bootstrapped when emcc needs it, and currently does not check if another process running emcc is bootstrapping it at the exact same time. If that occurs, there can be errors with `bootstrapping relooper failed. You may need to manually create src/relooper.js by compiling it`.

This can only happen when compiling to JS in parallel, not when compiling to bitcode.

**Comment by juj** [comment]
Relooper is no longer executed as a .js file, so the race condition doesn't occur there anymore. However this bug is kind of superseded by races in other locations, i.e. bugs #3850 and #4151. Closing as old/duplicate.
